MySQLOO/source/GMModule.cpp

```diff
@@ -7,7 +7,13 @@
 #define MYSQLOO_VERSION "9"
 #define MYSQLOO_MINOR_VERSION "6"
 
+// Variable to hold the reference to the version check ConVar object
+int iVersionCheckConVar = NULL;

```

Comment:
Please rename this to camelCase rather than hungarian notation, we use camelCase everywhere else in this project.

Reply:
Other than that it looks really good.

Reply:
Sure thing, I'll make the changes now 👌